docs/usage/user-stories/swissquote.md

```diff
@@ -0,0 +1,235 @@
+
+## The future of Renovate at Swissquote
+
+Not all teams are using Renovate at this stage, as some teams prefer to manually update their dependencies.
+We would like to enable Renovate for critical dependencies in all repositories and hope to make it useful and easy so other teams will adopt it for more dependencies.
+
+## How should I get started with Renovate?
+
+If this article convinced you, how should you get started?
+
+1. First, if you know your software is very outdated; don’t enable Renovate right away, you will be swamped with Pull Requests, we’ve been there and it’s not a happy memory. First take the time to **manually upgrade all that you can**. `npm outdated`, `mvn versions:display-dependency-updates` or your package manager’s equivalent of that command can help you get started, test your application and commit that
+1. You can now enable Renovate and will receive a Pull Request to add a configuration file, **read this first PR carefully** as it will explain what kind of PRs you are going to receive and when
+1. Make sure to **pick a schedule**, otherwise you will receive PRs at any time of the day. Our team schedules all third party dependencies on the weekend, automatically merging when the tests pass and we investigate the failing ones on Monday morning
+1. **Group Pull Requests**, if every PR has to go through CI, it can become quite heavy, once most PRs succeed, you can start grouping minor and patch updates so that you get a single PR per repository. Investigating issues becomes a bit trickier because of that. To give you some numbers out of 90 repositories enabled with Renovate, we have on average 4 PRs to investigate per week. Everything else is automatically merged
+1. **Enable auto merging after some time**, make sure your tests are solid and that you won’t upgrade something that wasn’t tested and will break once in production
+
+## Is all this effort worth it?
+
+The short answer is yes!
+
+It took us almost a year, way longer than we thought, to catch up with the latest version of everything in our technical stack.
+But once we were confident enough in our tests to enable automatic merging of Pull Requests we were satisfied of the work we did to update our software stack.
+We knew we would be ready when an unplanned change arrives.
+
+When that day came, in December 2021 with Log4Shell, it was a matter of hours to release the freshly merged Pull Requests, deploy the few applications we had, and notify the teams depending on our libraries.
+We were so quick to do it, in fact, that we had to do that three times with the chain of vulnerabilities that was found in Log4j that week.
+
+Keep in mind though that keeping your dependencies up-to-date is not just about the tooling ([Renovate](https://www.mend.io/free-developer-tools/renovate/), [Dependabot](https://github.com/features/security), [Snyk](https://snyk.io/product/open-source-security-management/), ...), you will find a tool that fits your use case.
```

Comment:
@rarkins The article mentions **Dependabot** and **Snyk**. Do you want to keep those references?

Reply:
I thought about this my 2 cents on the topic is that I prefer when a tool mentions its alternatives, the user can decide which they prefer. I find this approach more fair, and shows more confidence in your tool.

Reply:
I don't mind in theory, and especially not on your own Medium article, but I also don't think that this sentence adds anything. It's also particularly confusing to any reader to have this sentence on Renovate's own documentation page because they'll wonder if *we* are suggesting they consider alternatives. I would prefer to remove it as-is due to this aspect (it adds little value, but potentially adds confusion)

Reply:
```suggestion
```

Reply:
Sure, I understand for the confusion aspect